### ISSUE TITLE
Translation-related discussion

### ISSUE THREAD

**Issue by harisont** [harisont/covid-19#3]
Feel free to make suggestions or ask questions about controversial terms and expressions down here.

**Comment by pietroppeter** [comment]
There are a lot of terms that appear all over the place and we should have if possible a uniform translation. One term that I am having second thoughts after doing some translation:

* face masks: currently I translated with "maschere" (I do not think in italian we should specifiy they are for the face) but probably a better one would be "mascherine" (I see it as more used looking around in italian news site). A more specific could be "maschere protettive".

**Comment by mendaomn** [comment]
> * face masks: currently I translated with "maschere" (I do not think in italian we should specifiy they are for the face) but probably a better one would be "mascherine" (I see it as more used looking around in italian news site). A more specific could be "maschere protettive".

I can confirm it's mostly referred to as "mascherine", even though Wikipedia calls them "mascherine chirurgiche"

> Una maschera (o mascherina) chirurgica, nota anche come maschera medica o maschera facciale per uso medico

(source: https://it.wikipedia.org/wiki/Maschera_chirurgica)

**Comment by harisont** [comment]
> > * face masks: currently I translated with "maschere" (I do not think in italian we should specifiy they are for the face) but probably a better one would be "mascherine" (I see it as more used looking around in italian news site). A more specific could be "maschere protettive".
> 
> I can confirm it's mostly referred to as "mascherine", even though Wikipedia calls them "mascherine chirurgiche"
> 
> > Una maschera (o mascherina) chirurgica, nota anche come maschera medica o maschera facciale per uso medico
> 
> (source: https://it.wikipedia.org/wiki/Maschera_chirurgica)

Yes. And even in colloquial language I always heard "mascherine"

**Comment by pietroppeter** [comment]
another issue with my translation we were discussing in the pull request is how to translate "playable simulations". I did transalte "simulazioni giocabili" consciously traying to retain some of the playfulness of the original, although in Italian this sound probably mych more playful with respect to the original. A better (more standard) option might be "simulazioni interattive" (I did decide for "giocabili" thinking that the original could also have used "interactive" and decided not to). I think we can change that too if others prefer also. Your thoughts?

Note that there might be a few places to change to keep consistency (e.g. 30 mins read/play -> leggi/gioca).

**Comment by harisont** [comment]
> another issue with my translation we were discussing in the pull request is how to translate "playable simulations". I did transalte "simulazioni giocabili" consciously traying to retain some of the playfulness of the original, although in Italian this sound probably mych more playful with respect to the original. A better (more standard) option might be "simulazioni interattive" (I did decide for "giocabili" thinking that the original could also have used "interactive" and decided not to). I think we can change that too if others prefer also. Your thoughts?

I think "simulazione interattiva" sounds more natural ("giocabile" alway sounds like a very literal translation from English that does not even take into account the variety of meanings "play" can take in English). 

> Note that there might be a few places to change to keep consistency (e.g. 30 mins read/play -> leggi/gioca).

This is maybe a bit harder to translate if we go for "simulazione interattiva". I would propose something non literal, for instance "durata dell'esperienza: 30 min" or even just "tempo di lettura: 30 min".

**Comment by harisont** [comment]
Also I have one doubt: the authors often use the acronym ICU, but we don't have a corresponding one, right?
Like, I've seen "Terapia Intensiva" capitalized occasionally, but never "TI" alone.

**Comment by pietroppeter** [comment]
I agree we should replace the acronym with "Terapia Intensiva".

**Comment by harisont** [comment]
I think it's pretty obvious that sentences repeated in many parts of the text should be translated identically throughout the article, so wrt 

> Play with this calculator to see how...

I was looking at @pietroppeter's translation and found that "calculator" = "calcolatore". I'm not 100% sure this term sounds right (because it means "computer" in many contexts), but I can't think of a good alternative ("simulatore" maybe? Or maybe even better "simulazione"?). Suggestions?

**Comment by pietroppeter** [comment]
mmh, calculator could also be "calcolatrice" ("calcolatore" is more old fashioned for computer, "calcolatrice" is more the kind every shop owner uses to compute a discount).  ("gioca con questa calcolatrice di R0..."), in this case it is not much of a simulator.

**Comment by harisont** [comment]
Yeah maybe "calcolatrice"

**Comment by harisont** [comment]
I think there is a misunderstanding, but I am not sure who is the one who misunderstands @pietroppeter: my impression is that "current cases" should be translated as "casi attivi" and not as "casi attuali", because in Italian we use "attuale" only to indicate that something is happening right now, while in sentences like

> Due to the extra interventions, current cases peak *before* herd immunity is reached.

"current" is meant more like "concurrent", in that the cases are simultaneous "to each other".

But again I dunno if I am right. Opinions?

**Comment by pietroppeter** [comment]
Yes definitely “casi attivi” is a better translation! I looked for other occurrences of current and I would say almost all could use “attivo”. There is only a “current reproduction number”. I think a possible translation of that could be “numero di riproduzione effettivo”.

**Comment by harisont** [comment]
> Yes definitely “casi attivi” is a better translation! 

I will change that on the master branch.

> I looked for other occurrences of current and I would say almost all could use “attivo”. There is only a “current reproduction number”. I think a possible translation of that could be “numero di riproduzione effettivo”.

That's probably right, but to be honest I dont know how much that is a technical term (maybe it has a standard translation). If someone know, please answer this.

**Comment by pietroppeter** [comment]
It is referenced in a few places as that, e.g.: https://www.prismamagazine.it/2018/10/28/vaccini-matematica-epidemie/

**Comment by harisont** [comment]
Then I'll change it right away

**Comment by pietroppeter** [comment]
Regarding the translation of "Physical Distancing" (see previous discussion [here](https://github.com/harisont/covid-19/pull/6#discussion_r419374804)), I think we should prefer "distanziamento fisico" here instead of "distanziamento sociale". It is correct that in the media you hear more "distanziamento sociale", but I think the authors are probably consciously trying to make the distinction. In other parts of the document in fact they refer to both: "Every COVID-19 intervention you've heard of – handwashing, social/physical distancing".

Also note that you can find around some explanations of the differences between the two, like here: https://cordis.europa.eu/article/id/415848-trending-science-move-over-social-distancing-it-s-time-for-physical-distancing/it

**Comment by harisont** [comment]
@pietroppeter you have a point!

**Comment by pietroppeter** [comment]
> It is referenced in a few places as that, e.g.: https://www.prismamagazine.it/2018/10/28/vaccini-matematica-epidemie/

I also found some references translating as "numero di riproduzione netto". Probably both are fine ("effettivo" and "netto") but a relevant reference which could favor the use of "netto" is this bullettin from ISS (Istituto Superiore di Sanità) which is supposed to come out weekly to report the R value of different regions (next issue should be due tomorrow). See page 20 at this link:

https://www.epicentro.iss.it/coronavirus/bollettino/Bollettino-sorveglianza-integrata-COVID-19_28-aprile-2020.pdf

It might be a small change that we do when we finalize and polish.

**Comment by harisont** [comment]
As mentioned in #8, we need to find a good translation for "circuit-breaker lockdown" (it's an expression used in several places). Waiting for your suggestions.

**Comment by pietroppeter** [comment]
yeah, I saw that, that's a tough one. Circuit breaker is "interruttore" but "lockdown a interruttore" does not sound very nice or evocative. A more liberal but quite figurative translation could be "lockdown spezza-crescita".

At the moment I have not found any reference in Italian that could suggest a translation. A phase were we have a series of lockdowns I think it could be actually translated as "lockdown intermittente" as already suggested (which can be found in newspaper references and it is immediately understandable).

The role of the lockdown as explained in the article is anyway always (the first and all successive ones) that of a "reset circuit breaker: you hit it and you can start again to test, trace, treat...

I am not sure why they use the specific terminology and I would have to review better the original article (also, in english is this standard terminology or is it something they came up with? If so that would point us in the direction of having anyway to come up with an original translation such as "spezza crescita" or similar...).

**Comment by pietroppeter** [comment]
> (also, in english is this standard terminology or is it something they came up with? If so that would point us in the direction of having anyway to come up with an original translation such as "spezza crescita" or similar...).

It appears it is terminology used to describe the April lockdowns in Singapore (not made up by authors but not so established anyway): https://en.wikipedia.org/wiki/COVID-19_pandemic_in_Singapore

Could not find articles in Italian providing a translation (searching with Singapore keywords).

**Comment by harisont** [comment]
Doubt:
I was sure that

> setting the transmission/recovery rates directly

in "chapter" "The last few months" meant "impostare direttamente _le velocità di trasmissione/guarigione_" because, even though "transmission/recovery" looks like a ratio but that makes less sense in the context (please look at the context) __and__ because "rates" is a plural, but after seeing @pietroppeter's translation 

> impostare direttamente un tasso di trasmissione/guarigione

I am not sure any longer.
Tell me what your understanding of this sentence is, if we don't get to a conclusion I can take a look at some other translation.

**Comment by pietroppeter** [comment]
Hi @harisont, sorry today could not put too much work in, I will catch up tomorrow.

> > setting the transmission/recovery rates directly
> 
> in "chapter" "The last few months" meant "impostare direttamente _le velocità di trasmissione/guarigione_" because, even though "transmission/recovery" looks like a ratio but that makes less sense in the context (please look at the context) **and** because "rates" is a plural, but after seeing @pietroppeter's translation
> 
> > impostare direttamente un tasso di trasmissione/guarigione
> 
> I am not sure any longer.
> Tell me what your understanding of this sentence is, if we don't get to a conclusion I can take a look at some other translation.

My understanding is that the authors refer to two specific rates, the transmission rate (how many infected per day) and the recovery rate (how many recovered per day), which are mentioned just before in the footnote. The translation I gave was supposed to mean something like "impostare direttamente un tasso di trasmissione o un tasso di guarigione". I see it can be misleading because of the '/' and I approve the better translation you are giving.

To be perfectly honest, that specific footnote is not one I particularly appreciate by the authors (but that has nothing to do with the translation and I definitely love the article overall): they are trying to make an important point (the simulation are very simplified because a lot of things are left out), but the specific point they make is about a sort of subtlety (instead of adding a +1 every X day I add a fraction every single day). Also the overall wording is somewhat complicated (I had to introduce in the translation "quello che dici al simulatore e quello che fa" to try to make it more understable). And I do not see a particular need in that context to mention the recovery rate that will be introduced much later (the point does not need that at all).

I am saying all the above because: 1) maybe I am not really getting the whole footnote and so better not take my remarks too much in consideration 2) if instead I got the gist of the footnote right, then the footnote itself was not very clear to start with so we might not worry too much about making it worse with a translation. 😛

**Comment by pietroppeter** [comment]
Hi all, I have an overdue task to share in this issue the proposal to change the main title of the article from "E adesso cosa succede?" to "E ora che succede?" (I think it is more direct and reflects more the style of the article). It is something I am proposing in #12 and I would like to hear your thoughts on this (@harisont, @mendaomn, @simonefalcini) and possibly others if they want to start joining the work and enjoying the discussion on this or other matters.

Overall for coordination the plan for me is to go on reviewing the overall translation and propose stuff in #12 but I did not have much time this last few days so if I do not get to it one of these evenings I should be able to work on it on Sunday. Also I plan to keep monitoring other work (such as the ongoing review of @harisont in #14 and trying to give feedback).

**Comment by harisont** [comment]
> Hi all, I have an overdue task to share in this issue the proposal to change the main title of the article from "E adesso cosa succede?" to "E ora che succede?" (I think it is more direct and reflects more the style of the article). It is something I am proposing in #12 and I would like to hear your thoughts on this (@harisont, @mendaomn, @simonefalcini) and possibly others if they want to start joining the work and enjoying the discussion on this or other matters.

Unlike in many other cases, I don't have a strong opinion about this. Probably my favorite translation would be "E adesso che (cosa) succede?", but all of them work and none sounds too formal to me.

**Comment by grgfgh** [comment]
Hi everyone, thank you for the great work you're doing!
I just wanted to let you know that I noticed that sometimes there's a trailing "s" after some icons to indicate the plural:
![image](https://user-images.githubusercontent.com/59988980/81212826-74c91d80-8fa3-11ea-8ca0-09d2218b9e02.png)
I'm not really familiar with github so I don't really know how to help you out without making a mess :) (I'm not even sure this is the right place to write you this comment)

sorry for the intrusion and keep up with the good work!

**Comment by pietroppeter** [comment]
Hi @grgfgh thanks for noticing this (and the wrong link to the thumbnail), we can fix those but if you want to give a try on making a PR (pull request) feel free to give it a try. If you Google “how to make a Pull request on github” you will find a lot of material and we can help you if you have questions. Otherwise you can keep giving good suggestions and we will take care of the PRs in the next few days!

**Comment by harisont** [comment]
I think in Italian we say both "infettiva" and "contagiosa" of a "malattia" but more commonly "contagiosa" when we refer to a person. Is that right?

**Comment by pietroppeter** [comment]
Yes, I think you are right on that point. We can maintain the wordplay by saying “sei contagiato ma non contagioso”. (Adding the emphasis over whole words or just on endings. Not able at the moment (from mobile) to comment directly on commit but I was noting also in the original commit a span tag around one of the icon we probably should keep (not sure why only for one, possibly because of commas).

**Comment by pietroppeter** [comment]
> > Hi all, I have an overdue task to share in this issue the proposal to change the main title of the article from "E adesso cosa succede?" to "E ora che succede?" (I think it is more direct and reflects more the style of the article). It is something I am proposing in #12 and I would like to hear your thoughts on this (@harisont, @mendaomn, @simonefalcini) and possibly others if they want to start joining the work and enjoying the discussion on this or other matters.
> 
> Unlike in many other cases, I don't have a strong opinion about this. Probably my favorite translation would be "E adesso che (cosa) succede?", but all of them work and none sounds too formal to me.

Ok, I went down the rabbithole of this what-a-great-idea-would-be-to-propose-to-change-the title-that-appears-in-tons-of-places-with-a-mostly-identical-wording and I am determined to see through it (and never do it again).

What I did is submit to a chat a friends (let's call them "Theatre People") to ask for advice. I received from those generous people supporting evidence for basically every other variation of the sentence and some. Here is an abridged summary of the discussion for your enjoyment:

On choosing between "E adesso cosa succede?" and "E ora che succede?":

- Io non vedo una grande differenza, nel lessico. “E adesso” è il sintagma che evoca la prosecuzione della trama/puntata. C’è una lievissima intertestualità, non so se voluta, ma che secondo me è proprio un lapsus linguistico/marca culturale. E adesso musica. E adesso pubblicità. Etc. E' il sintagma dell'aspettativa
- Io userei "E ora che succede?" perché ha meno lettere, ha meno difficoltà di pronuncia (adesso, cosa e succede hanno troppe esse) e risulta più incisiva.

New proposals:

- "Dunque?"
- "E mo'?" [rubato alla traduzione marchigiana in progress, ma anche lì sono indecisi con l'alternativa "Che d'è?"]
- "E adesso che succede?" [this received a +1]

On the alternative proposed by @harisont "E adesso che cosa succede?":

- Io la preferisco. Anche nella forma « E  ora/adesso cosa succede? ». Preferirei forse il «cosa » perchè il « che » da solo è più ambiguo in lettura, meno immediatamente chiaro, infatti forse un bambino (non toscano!) direbbe più così, « e ora cosa succede? » piuttosto che « e adesso che succede? »

And by the way I am noticing now that the thumbnail translation is (that @simonefalcini offer to change when we make up our mind) is "cosa succederà dopo?".

In the end I think that the result of the external survey is that basically any translation is fine*, so for the moment I think it will be more important to complete the rest of the review and maybe come back later and in the meantime we will have a few dozens other suggestions* to pick from.

*although I have to admit that "sintagma dell'aspettativa" made quite an impression on me.
**one of which could be: let's use a different translation for every place where the title appears! 😜

**Comment by harisont** [comment]
Again, I don't have any strong feelings about these expression in particular, even though this is indeed a kind of linguistic discussion I generally find interesting and your comment is definitely worth reading ;)
So I leave the decision to you @pietroppeter (because you seem to be the one who cares the most) **e non vedo l'ora di leggere questo articolo tradotto in marchigiano** (e magari anche altri dialetti...).

**Comment by harisont** [comment]
But there _is_ something I have strong feelings about: even in an informal text, I find it weirdly annoying to find occurrences of the so-called "tu generico" in Italian (in English it's so common that it does not bother me). So I am changing sentences like "quanto tempo resti Esposto" to the impersonal equivalent, e.g. "quanto tempo si resta Esposti".

If anyone has equally strong feelings about this, let me know before I do it too many times.

**Comment by harisont** [comment]
Also: both @pietroppeter and I translated "interventions" as "interventi", but I don't necessarily think it's a context-specific enough translation.
Should we change it to "interventi di contenimento" or "misure preventive" or something even better?

**Comment by harisont** [comment]
And also: I don't fully understand footnote `exact_formula` nor its translation.
